02-express-tutorial/app.js

```diff
@@ -1,8 +1,41 @@
 const express = require('express')
 const app = express()
-const { products } = require("./data");
+const { products } = require("./data")
+const peopleRouter = require("./routes/people")
+
+const logger = (req, res, next) => {
+    time = new Date()
+    console.log(`Method: ${req.method} URL: ${req.url} Time: ${time.getHours()}:${time.getMinutes()}:${time.getSeconds()}`)
+    next()
+}
+
+app.use('/', logger)
+app.use(express.urlencoded({ extended: false }))
+app.use(express.json())
+app.use("/api/v1/people", peopleRouter);
+
+// app.use(express.static("./public"))
+
+// app.get('/', logger, (req, res) => {
+//     res.send('<h1>OK</h1>')
+// })
```

Comment:
Because the requirements ask to use tow approaches of some methods, so I commented out the first approach to let you know that I use the first approach, that's it, I will clear all comments now

Reply:
Sure, that makes sense! Especially for personal projects and for study.